### ISSUE TITLE
syntax highlighting for "Data to send with request" field

### ISSUE THREAD

**Issue by ahillio** [Webskillet/rules_api#2]
Syntax highlighting for the field *Data to send with request* would be a nice touch.  dgo.to/ace_editor might be useful for that.

**Comment by jkissam** [comment]
agreed, I just haven't had the time to get to this, though if anyone wants to pay me to integrate it I would prioritize it (and it should be pretty quick/cheap)

**Comment by ahillio** [comment]
It's a very minor request, but I figured I'd add it here with the other to have it on record :)
